modules/swagger-codegen/src/main/resources/csharp/README.mustache

```diff
@@ -126,6 +128,7 @@ No model defined in this package
 
 ## Documentation for Authorization
 
+<a name="{{#authMethods}}{{name}}"></a>
```

Comment:
This line is causing issue due to missing close tag `{{/authMethods}}` and I've removed it in my PR.